tests/baselines/reference/nodeModulesTripleSlashReferenceModeDeclarationEmit5(module=node12).js

```diff
@@ -0,0 +1,38 @@
+//// [tests/cases/conformance/node/nodeModulesTripleSlashReferenceModeDeclarationEmit5.ts] ////
+
+//// [package.json]
+{
+    "name": "pkg",
+    "version": "0.0.1",
+    "exports": {
+        "import": "./import.js",
+        "require": "./require.js"
+    }
+}
+//// [import.d.ts]
+export {};
+declare global {
+    interface ImportInterface {}
+}
+//// [require.d.ts]
+export {};
+declare global {
+    interface RequireInterface {}
+}
+//// [index.ts]
+/// <reference types="pkg" resolution-mode="import"/>
+/// <reference types="pkg" resolution-mode="require"/>
+export interface LocalInterface extends ImportInterface, RequireInterface {}
+
+//// [index.js]
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+/// <reference types="pkg" resolution-mode="import"/>
+/// <reference types="pkg" resolution-mode="require"/>
+
+
+//// [index.d.ts]
+/// <reference types="pkg" resolution-mode="import"/>
+/// <reference types="pkg" />

```

Comment:
I'm still a bit bothered by omitting something that people wrote, even if it's redundant. Do we do this elsewhere?

Edit: Oh, it's because of backward compatibility, isn't it -- we don't want to start emitting `resolution-mode="require"` in all d.ts files everywhere.

Reply:
Yeah, while there are upsides to making `resolution-mode="require"` explicit (being able to move the file between cjs/esm contexts and needing to change less), it's ugly and there are also downsides (it's behavior not changing to match the context it's moved to, as may be expected).